blog/views.py

```diff
@@ -17,15 +21,30 @@
 #     # queryset = Entry.objects.published()
 #     template_name = "home.html"
 #     paginate_by = 2
-class EntryView(ListView):
-    template_name = 'Entry/Entry.html'
+class EntryView(DetailView):
+    template_name = 'blog/view_entry.html'
+    model = Entry
+    pk_url_kwarg = 'entry_id'
+
+    def get_context_data(self, **kwargs):
```

Comment:
since this method isn't actually doing anything here it can be removed. I just added it in to show you how you could debug.  You could also add in additional variables to your context if you wanted, but you don't really need to do that here.